From 182ac066474e579b0548fdd4a5698c8c13a73d50 Mon Sep 17 00:00:00 2001
From: Mingcong Bai <jeffbai@aosc.io>
Date: Fri, 2 Oct 2026 00:37:53 +0800
Subject: [PATCH 2/2] AOSCOS: mainwindow: enlarge default window size to
 1024x600

It's not too much, right? The default size was way too small and slender
for some reason...

Signed-off-by: Mingcong Bai <jeffbai@aosc.io>
---
 src/mainwindow.ui | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mainwindow.ui b/src/mainwindow.ui
index de666a6f..2b416a81 100644
--- a/src/mainwindow.ui
+++ b/src/mainwindow.ui
@@ -6,8 +6,8 @@
    <rect>
     <x>0</x>
     <y>0</y>
-    <width>779</width>
-    <height>362</height>
+    <width>1024</width>
+    <height>600</height>
    </rect>
   </property>
   <property name="sizePolicy">
-- 
2.55.0

